docs/features/logging.md

```diff
@@ -45,99 +47,6 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 If you have multiple middleware components configured, it is advised to put the `ExceptionHandlingMiddleware` as the first one.
 By doing so, unhandled exceptions that might occur in other middleware components will also be logged by the `ExceptionHandlingMiddleware` component.
 
-## Logging incoming requests
```

Comment:
Why are we removing this here? Because it's moved to preview?

Reply:
Yes, we have not yet released a version where this is available.